bundler-packages/resolutions-rs/benchmark/mocked-js-impl.js

```
@@ -0,0 +1,90 @@
+        return `${remappings[prefix]}${importPath.substring(prefix.length)}`;
+      }
+      
+      // Try libs as a fallback
+      for (const lib of libs) {
+        const potentialPath = `${lib}/${importPath}`;
+        return potentialPath; // Return first match
+      }
+      
+      // Return the import path as is if we can't resolve it
+      return importPath;
+    }
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)